Docs/Specifics/Choreo.md

```diff
@@ -0,0 +1,68 @@
+# Choreo
+
+Choreo is a path planning library that uses path-optimization to create time-constrained autonomous paths for the autonomous period. Learning to integrate Choreo into our codebase is important as autonomous, the first 15 seconds of the match, can be the make or break of a high-scoring teleop match.
+
+## Choreo Structure
+
+The following is the stack that is involved to integrate with Choreo:
+
+### Choreo GUI
+
+![Choreo Hero Home Page](/Assets/choreo-hero.png)
+
+The Choreo GUI is the multiplatform Tauri (React + Rust) app that allows you to create and save the paths. It's an independent app from Visual Studio or WPILib (although for the 2025 season it might be bundled in WPILib).
+
+### [Sleipnir](https://github.com/SleipnirGroup/Sleipnir)
+
+Sleipnir, written in C++23, is a "a linearity-exploiting sparse nonlinear constrained optimization problem solver that uses the interior-point method." What this really means is that based on multiple constraints and variables (some listed [here](https://sleipnirgroup.github.io/Choreo/document-settings/robot-configuration/)), the problem-solver is able to constrain them while optimizing the path time. 8033 was the first team to run Choreo at a competition (Chezy Champs, where we saw very reliable paths and great potential in constrast to what we used to use: PathPlanner). 
```

Comment:
see above, the explanation of sleipnir can be extended a little bit or link to another resource to introduce the concept and *why* trajopt is useful, as well as comparing it to splines.